FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Oleksii Levchenko on 13.09.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal enum FeedItemsMapper {
+	private struct Root: Decodable {
+		private let items: [Image]
+
+		var feed: [FeedImage] { items.map(\.feedImage) }
+	}
+
+	private struct Image: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
+
+		enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+
+		var feedImage: FeedImage {
+			FeedImage(
+				id: id,
+				description: description,
+				location: location,
+				url: url
+			)
+		}
+	}
+
+	internal static func map(data: Data, response: HTTPURLResponse) -> FeedLoader.Result {
```

Comment:
Same as the previous comment. You can consider removing `internal` here.
```suggestion
	static func map(data: Data, response: HTTPURLResponse) -> FeedLoader.Result {
```